components/omega/test/ocn/AuxiliaryVarsTest.cpp

```diff
@@ -621,6 +670,92 @@ int testVelocityDel2AuxVars(Real RTol) {
    return Err;
 }
 
+int testTracerAuxVars(const Array2DReal &LayerThickCell,
+                      const Array2DReal &NormalVelEdge, Real RTol) {
```

Comment:
Add some comment that we're only testing on the planar mesh (if I understand correctly)?

Reply:
Tests are done on both planar and spherical meshes. The build creates separate executables for planar or spherical tests that open the desired mesh: file https://github.com/brian-oneill/Omega/blob/c67850bacee510ec744dcc6e6e7f6edb3adf99b0/components/omega/test/ocn/AuxiliaryVarsTest.cpp#L277-L285

Reply:
Cool! Thanks